app/assets/javascripts/data_grid.js.coffee

```diff
@@ -73,22 +107,22 @@ window.configs =
       [
         targets: [3]
```

Comment:
Would it be better if I attach a column_name_column html class to each column and then change that js declaration into: `tagets: 'column_name_column'` ?